src/ipfs/fetchIpfs.ts

```diff
@@ -0,0 +1,12 @@
+export async function fetchIpfs<T>(ipfsUrl: string): Promise<T> {
+    const url = ipfsToHttps(ipfsUrl)
+    const fetchRes = await fetch(url);
+    const fetchJson = await fetchRes.json();
+    return fetchJson as T
+}
+
+export function ipfsToHttps(ipfsUrl: string) {
```

Comment:
There is already a utility function that handles this in `src/SongList/SongList.utils.ts`. 

Seeing as this utility function might be used in multiple place. I propose you create it under the following file structure `src/services/Ipfs/service.ts`, and then you export all the functions needed from that module.

The proposed structure will leave room for writing test cases for that utility in the future in the following file `src/services/Ipfs/test.ts`